src/Models/DataType.php

```diff
@@ -55,6 +55,31 @@ public function deleteRows()
         return $this->rows()->where('delete', 1);
     }
 
+    public function addRelationshipRow($requestData)
```

Comment:
It simply adds the relationship field to data_rows table. Do you think it belongs in DataRow?